container/ubuntu16_04/bazel/container.yaml

```diff
@@ -22,13 +23,22 @@ commandTests:
 - name: 'check-openssl'
   command: ['bash', '-c', 'openssl version']
   expectedOutput: ['OpenSSL .*']
-- name: 'bazel'
-  command: ['bazel']
-  expectedOutput: ['Usage: bazel <command> <options>.*']
+- name: 'check-git'
+  command: ['bash', '-c', 'git --version']
+  expectedOutput: ['git version .*']
+- name: 'check-unzip'
+  command: ['bash', '-c', 'unzip']
```

Comment:
add a check that container has gcc installed

Reply:
Done.